extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItem.java

```diff
@@ -81,12 +81,12 @@ public void setUploadDate(@Nullable DateWrapper uploadDate) {
         this.uploadDate = uploadDate;
     }
 
-    public int getLikeCount() {
-        return likeCount;
+    public String getTextualVoteCount() {
+        return textualVoteCount;
     }
 
-    public void setLikeCount(int likeCount) {
-        this.likeCount = likeCount;
+    public void setTextualVoteCount(String textualVoteCount) {
+        this.textualVoteCount = textualVoteCount;
```

Comment:
Please change the public API through deprecation instead of just outright removing it.

Reply:
Generally, yes, but in this case the API should not change (textualLikeCount can be added, but likeCount remains). `Utils.mixedNumberWordToLong()` should do the job.